hc/accounts/tests/test_profile.py

```diff
@@ -41,10 +46,13 @@ def test_it_adds_team_member(self):
             member_emails.add(member.user.email)
 
         ### Assert the existence of the member emails
+        self.assertIsNotNone(member_emails)
 
         self.assertTrue("frank@example.org" in member_emails)
```

Comment:
you can improve this test by writing `self.assertTrue(form["email"] in member_emails`